sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php

```diff
@@ -1255,4 +1329,18 @@ public static function isAssoc(array $arr = NULL) {
     return array_keys($arr) !== range(0, count($arr) - 1);
   }
 
+  /**
+   * Check if this is eReolen GO.
+   */
+  public static function isBreol(): bool {
+    return module_exists('breol_news');
```

Comment:
Do this trigger an file system search? If so cache the result...it will not change on a given site. Static cache?

Reply:
[`module_exists`](https://api.drupal.org/api/drupal/includes%21module.inc/function/module_exists/7.x) depends on [`module_list`](https://api.drupal.org/api/drupal/includes%21module.inc/function/module_list/7.x) which handles caching.